intakebuilder/getinfo.py

```diff
@@ -109,8 +90,7 @@ def getInfoFromGFDLDRS(dirpath,projectdir,dictInfo):
    #               "ensemble_member", "grid_label", "variable",
    #               "temporal subset", "version", "path"]
  
-#/archive/oar.gfdl.cmip6/ESM4/DECK/ESM4_historical_D1/gfdl.ncrc4-intel16-prod-openmp/pp/atmos/ts/monthly/5yr/DO_NOT_USE/atmos.201001-201412.alb_sfc.nc
-
+#Grab values based on their expected position in path 
```

Comment:
builderconfig did this, without harcoding "ts" and other expectations in the code.
the following got info from the builderconfig file with an understanding that the last one is filename. From there on, it 
 dictInfo[builderconfig.output_file_template[nlen-i]] = ncfilename[-i]

see  section in getinfo.py below-- 
   #lets go backwards and match given input directory to the template, add things to dictInfo

Reply:
Yes, I agree we may need to return this functionality